servlet/security-clientcert/src/main/java/org/javaee7/servlet/security/clientcert/SecureServletWithCommonName.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.javaee7.servlet.security.clientcert;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ *
+ * @author cuba
```

Comment:
Nicer to have a full name: get your name out there! :)

Reply:
Can update IDE settings. It's probably pulling linux username